test/e2e/csv_e2e_test.go

```diff
@@ -1015,7 +1015,7 @@ var _ = Describe("CSV", func() {
 		require.True(GinkgoT(), ok, "expected olm sha annotation not present on existing pod template")
 
 		// Induce a cert rotation
-		Eventually(Apply(fetchedCSV, func(csv *v1alpha1.ClusterServiceVersion) error {
+		Eventually(ctx.Ctx().ServerSideApplier().Apply(fetchedCSV, func(csv *v1alpha1.ClusterServiceVersion) error {
```

Comment:
Could we make this read nicer by having an `Apply` method on `ctx.Ctx()` or performing this access in a top-level `Apply` function?